src/cf-layout.less

```diff
@@ -1404,6 +1404,289 @@
+*/
+
+@hero-desktop-height: 300px;
+@hero-image-height: 195px;
+@desktop-image-container-width: 475px; //width of 5 of 12 columns at 1140px
+@tablet-image-container-width: 333px; //width of 5 of 12 columns at 799px
+
+.hero {
+  background-color: @gray-10;
+  
+  &_wrapper {
+    position: relative;
+    padding-top: unit(@grid_gutter-width / @base-font-size-px, em);
+    // bottom padding = image height at mobile + padding for top/bottom of image
+    padding-bottom: unit((@hero-image-height + @grid_gutter-width * 2) / @base-font-size-px, em); 
+    
+    &:after {
+      display: block;
+      width: 100%;
+      height: 100%;
+      position: absolute;
+      top:0;
+      right: 0;
+      content: "";
+      background-position: center bottom unit(@grid_gutter-width / @base-font-size-px, em);
+      background-repeat: no-repeat;
+    }
+
+    .respond-to-min(@tablet-min, {
+      padding-right: unit(@tablet-image-container-width / @base-font-size-px, em);
+      padding-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
+      
+      &:after {
+        width: @tablet-image-container-width;
+        background-position: center;
+      }
+    });
+
+    .respond-to-min(800px, {
+      box-sizing: border-box;
+      min-height: @hero-desktop-height;
+      padding-top: unit(40px / @base-font-size-px, em); // approximates 45px visually
+      padding-right: unit(@desktop-image-container-width / @base-font-size-px, em);
+      padding-bottom: unit(40px / @base-font-size-px, em); // approximates 45px visually
+      
+      &:after {
+        width: @desktop-image-container-width;
+        background-position: center;
+      }
+    });
+    
+    .lt-ie9 & {
+      // Reduce min-height because IE8 apparently isn't honoring the border-box property,
+      // perhaps due to bug seen here: http://grab.by/JutE
+      min-height: 0;
+      // Fallback for four-value bg-pos syntax
+      &:after {
+        background-position: right center;
+      }
+    }
+  }
+  
+  &_heading {
+    .h2();
+    .respond-to-min(800px, {
+      .h1();
+    });
+  }
+  
+  &_subhead {
+    .h4();
+    .webfont-regular();
+    margin-bottom: 0;
+
+    .respond-to-min(800px, {
+      .h3();
+      margin-bottom: 0;
+    });
+  }
+
+  &_cta {
+    .webfont-regular();
+    margin-top: @grid_gutter-width;
+    margin-bottom: 0;
+    
+    a {
+      border-bottom-width: 1px;
+      cursor: pointer;
+    }
+  }
+  
+  // increase top padding to compensate for less text
+  &__small-content &_wrapper {
+    .respond-to-min(800px, {
+      padding-top: unit(55px / @base-font-size-px, em); // approximates 60px visually
+    });
+  }
+  
+  // if the image is wide, position at top and right
+  &__wide-image &_wrapper:after {
+    .respond-to-min(800px, {
+      background-position: top unit(40px / @base-font-size-px, em) right;
+    });
+  }
+}
+
+// simplifies customizing a hero
+.setup-hero(@url: '', @width:0, @height:0, @tablet-width:@width, @desktop-width:@width) {
```

Comment:
Should there be some defaults set above that this over-rides, like `width: 100%`. That way if we're using one set of sizes everywhere and only changing the image we're not repeating the same styles over and over?